packages/prover/src/cli/cmds/start/options.ts

```diff
@@ -54,16 +55,19 @@ export const startOptions: CliCommandOptions<StartArgs> = {
 
   beaconUrls: {
     description: "The beacon node PRC urls for 'rest' mode.",
-    type: "string",
-    array: true,
+    type: "array",
+    string: true,
+    coerce: (urls: string[]): string[] =>
+      // Parse ["url1,url2"] to ["url1", "url2"]
+      urls.map((item) => item.split(",")).flat(1),
```

Comment:
Depth is not necessary here as `map` & `split` combination will only generate 1 level of nested arrays. 

```suggestion
      urls.map((item) => item.split(",")).flat(),
```